examples/tests/kv/kv_unit_tests/Makefile

```diff
@@ -6,7 +6,7 @@ TOCK_USERLAND_BASE_DIR = ../../../..
 # Which files to compile.
 C_SRCS := $(wildcard *.c)
 
-ELF2TAB_ARGS += --write_id 17767 --read_ids 17767 --access_ids 17767
+ELF2TAB_ARGS += --write_id 1153320202 --read_ids 1153320202 --access_ids 1153320202
```

Comment:
Just curious, why these changes? If there's meaning in the magic numbers, might be worth documenting?

Reply:
As Tock no longer uses `write_id`s these numbers need to match the actual short ID generated in the kernel. Which generally is a SHA of the name. So this is updating these to match the numbers the kernel uses.

It's a pain to work with and very confusing unfortunately

Reply:
Gotcha. Mind adding a comment for this? I agree that this is confusing, esp. to someone uninitiated to the whole AppId subsystem, so I think a comment might help a lot!